Code/GraphMol/hanoitest.cpp

```diff
@@ -1561,49 +1559,9 @@ void testGithub1567() {
   BOOST_LOG(rdInfoLog) << "Finished" << std::endl;
 }
 
-void testCanonicalDiastereomers() {
-  // FIX: this is another one that we dno't currently handle properly
-#if 0
-  BOOST_LOG(rdInfoLog) << "testing diastereomer problem." << std::endl;
-
-  auto m1 = "F[C@@H](Cl)[C@H](F)Cl"_smiles;
-  auto m2 = "F[C@H](Cl)[C@@H](F)Cl"_smiles;
-  auto smi1 = MolToSmiles(*m1);
-  auto smi2 = MolToSmiles(*m2);
-  TEST_ASSERT(smi1 != smi2);
-  BOOST_LOG(rdInfoLog) << "Finished" << std::endl;
-#endif
-}
-
-void testRingsAndDoubleBonds() {
-// FIX: we don't currently handle this case properly
-#if 0
-  BOOST_LOG(rdInfoLog)
-      << "testing some particular ugly para-stereochemistry examples."
-      << std::endl;
-  std::vector<std::string> smis = {"C/C=C/C=C/C=C/C=C/C", "C/C=C1/C[C@H](O)C1",
-                                   "C/C=C1/CC[C@H](O)CC1"};
-  for (const auto smi : smis) {
-    SmilesParserParams ps;
-    ps.sanitize = false;
-    ps.removeHs = false;
-    std::unique_ptr<ROMol> mol(SmilesToMol(smi, ps));
-    TEST_ASSERT(mol);
-    mol->setProp(common_properties::_StereochemDone, 1);
-    mol->updatePropertyCache();
-    MolOps::setBondStereoFromDirections(*mol);
-    std::cerr << "   " << MolToSmiles(*mol) << std::endl;
-    _renumberTest(mol.get(), smi, 500);
-    std::cerr << "   " << MolToSmiles(*mol) << std::endl;
-  }
-  BOOST_LOG(rdInfoLog) << "Finished" << std::endl;
-#endif
-}
```

Comment:
Rather than deleting this, I would prefer to re-enable it with the one failing test commented out (I will create a separate issue for that):
```
void testRingsAndDoubleBonds() {
  BOOST_LOG(rdInfoLog)
      << "testing some particular ugly para-stereochemistry examples."
      << std::endl;
  std::vector<std::string> smis = {
      "C/C=C/C=C/C=C/C=C/C",  // "C/C=C1/C[C@H](O)C1",
      "C/C=C1/CC[C@H](O)CC1"};
  UseLegacyStereoPerceptionFixture reset_stereo_perception(false);
  for (const auto &smi : smis) {
    SmilesParserParams ps;
    ps.sanitize = false;
    ps.removeHs = false;
    std::unique_ptr<ROMol> mol(SmilesToMol(smi, ps));
    TEST_ASSERT(mol);
    mol->setProp(common_properties::_StereochemDone, 1);
    mol->updatePropertyCache();
    MolOps::setBondStereoFromDirections(*mol);
    std::cerr << "   " << MolToSmiles(*mol) << std::endl;
    _renumberTest(mol.get(), smi, 500);
    std::cerr << "   " << MolToSmiles(*mol) << std::endl;
  }
  BOOST_LOG(rdInfoLog) << "Finished" << std::endl;
}

```